Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateAVFoundationObjC.mm

```diff
@@ -2892,6 +2898,12 @@ void determineChangedTracksFromNewTracksAndOldItems(MediaSelectionGroupAVFObjC*
 
 #if ENABLE(LEGACY_ENCRYPTED_MEDIA)
 
+void MediaPlayerPrivateAVFoundationObjC::setShouldContinueAfterKeyNeeded(bool shouldContinue)
```

Comment:
not needed

Reply:
I added it for the logging, which helped me figure out the problem. I think the logging may be helpful in the future so I will leave it.